src/pudl/transform/ferc1.py

```diff
@@ -1196,9 +1320,13 @@ def merge_instant_and_duration_tables_xbrl(
                 )
                 out_df = pd.concat(
                     [
-                        instant.set_index(["report_year", "entity_id"] + instant_axes),
+                        instant.set_index(
+                            ["report_year", "entity_id", "sched_table_name"]
+                            + instant_axes
+                        ),
                         duration.set_index(
-                            ["report_year", "entity_id"] + duration_axes
+                            ["report_year", "entity_id", "sched_table_name"]
+                            + duration_axes
```

Comment:
can we not use the `instant_merge_keys` and `duration_merge_keys` for this for some reason that I'm not seeing? they look like they are the same and it would be nice to only need to update them in one place.

Reply:
gr8 point, I'll make it the keys